app/views/layouts/application.html.haml

```diff
@@ -10,15 +10,23 @@
     = javascript_pack_tag 'navbar'
     / If using inline styles in the vue pack, loading the stylesheet_pack_tag as well becomes necessary
     = stylesheet_pack_tag 'navbar'
+
+    = javascript_pack_tag 'project_members'
+    = stylesheet_pack_tag 'project_members'
+    = javascript_pack_tag 'users'
+    = stylesheet_pack_tag 'users'
+    = javascript_pack_tag 'projects'
+    = stylesheet_pack_tag 'projects'
```

Comment:
The line below yields to assets so you do not need to include these here. Please include in the haml for each specific page